framework/rbac/PhpManager.php

```diff
@@ -796,7 +796,7 @@ protected function loadFromFile($file)
      */
     protected function saveToFile($data, $file)
     {
-        file_put_contents($file, "<?php\nreturn " . VarDumper::export($data) . ";\n", LOCK_EX);
+        file_put_contents($file, "<?php\r\n\r\nreturn " . VarDumper::export($data) . ";\r\n", LOCK_EX);
```

Comment:
How about using \n without \r?

Reply:
now that is look at it it makes sense... should I make another pull-request?

Reply:
No. Just push another commit to the same branch.